transformation-builder/src/main/kotlin/com/cloudinary/transformation/layer/source/TextSource.kt

```diff
@@ -1,31 +1,30 @@
 package com.cloudinary.transformation.layer.source
 
-import com.cloudinary.transformation.Color
-import com.cloudinary.transformation.ITransformableImage
-import com.cloudinary.transformation.ImageTransformation
-import com.cloudinary.transformation.Param
+import com.cloudinary.transformation.*
```

Comment:
Is it considered a good practice to `import *` ?

Reply:
Fixed